apps/ui/src/Components/CommunityList/CommunityList.tsx

```diff
@@ -11,10 +12,11 @@ dayjs.extend(relativeTime);
 
 interface CommunityListProps {
   communities: CommunityWithRides[];
+  setSelectedRide: Dispatch<SetStateAction<Ride | undefined>>;
 }
 
 const CommunityList: React.FC<CommunityListProps> = (props) => {
```

Comment:
Consider changing the name to something more clear

Reply:
What do you mean here? What's not clear?